libraries/AP_HAL_ChibiOS/hwdef/CubeGreen-solo/hwdef.dat

```diff
@@ -9,3 +9,35 @@ include ../CubeBlack/hwdef.dat
 # pull Solo's default parameters from /Tools/Frame_params
 # these are parameters the Solo requires for proper operation that are differnet from the 3.7 standard defaults.
 env DEFAULT_PARAMETERS 'Tools/Frame_params/Solo_Copter-3.7_GreenCube.param'
+
```

Comment:
Why do you need this? Aren't these inherited from the CubeBlack include?

Reply:
fixed, thanks!